body.adoc

```diff
@@ -520,20 +525,19 @@ while executing in an enabled privilege mode, and when FROZEN=0.
 
 The elapsed cycle count value is represented by the CC field, which has
 a mantissa component (Cycle Count Mantissa, or CCM) and an exponent
-component (Cycle Count Exponent, or CCE). When CCE=0, CCM holds the raw
-number of elapsed cycles. When CCE>0, the cycle count can be calculated
+component (Cycle Count Exponent, or CCE). The elapsed cycle count can be calculated
 using the following formula:
 
 [subs="specialchars,quotes"]
 ----
-(2^12^ + CCM) << CCE-1
+CCE==0? CCM : ((2^12^ + CCM) << CCE-1)
 ----
 [WARNING]
 ====
 _The 2^12^ is just a way to represent the leading 1.  Let me know if there's a better way to describe this._
 ====
 
-When CCE>1, the cycle count granularity is reduced from 1 to 2^CCE-1^.
+When CCE<=1, including cases where 13 or fewer CC bits are implemented, the CC field provides a simple binary count of elapsed cycles.  When CCE>1, the cycle count granularity is reduced from 1 to 2^CCE-1^.

```

Comment:
Should this be "the CCM field"?

Reply:
Oops, meant to remove that text, that was an earlier attempts at clarifying the behavior.  I think the equation now covers everything, so will remove it.